app.py

```diff
@@ -8,27 +8,47 @@
 from flask import Flask, flash, request, redirect, render_template, url_for, send_from_directory
 from werkzeug.utils import secure_filename
 from converters.json2glm import json2glm
+from converters.json2png import json2png
 
 app = Flask(__name__)
 app.config.from_object("settings.config.DevelopmentConfig")
 
+supported_from_to_conversions = {
+    "json": {
+        "glm": json2glm,
+        "png": json2png,
+    },
+}
+default_json2png = {'output_type': 'summary', 'with_nodes': False, 'resolution': '300', 'size': '300x200', 'limit': None}
```

Comment:
Why not add this settings/default into `supported_from_to_conversions` dict?

My approach here was a suggestion. Please feel free to expand/improve/riff on it.

Reply:
My idea was to add a dictionary for each of the converters that take in arguments that have default values. Then all the dictionaries are collected in a dictionary called defaults in upload_file() which is used to replace any empty value. The reason for a dict for each converter is to be able to keep track of arguments and their defaults.

Reply:
> My idea was to add a dictionary for each of the converters that take in arguments that have default values. Then all the dictionaries are collected in a dictionary called defaults in upload_file() which is used to replace any empty value. The reason for a dict for each converter is to be able to keep track of arguments and their defaults.

Sure, but why not make it part of the conversions dict?
Something like:
```python
supported_from_to_conversions = {
    "json": {
        "glm": json2glm,
        "png": {
            "defaults": { "output_type": ...},
            "function_handler": json2png,
        },
    },
}
```

Reply:
Yeah, that's makes more sense. Thanks